Modules/Login/Entities/User.php

```diff
@@ -4,13 +4,25 @@
 
 use Illuminate\Database\Eloquent\Model;
 use Modules\Brand\Entities\Brand;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Spatie\Permission\Traits\HasRoles;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
-class User extends Model
+class User extends Authenticatable
 {
+    use HasRoles;
+    use SoftDeletes;
     protected $fillable = [];
 
+    const ROLE_BRAND = 'brand';
+    const ROLE_RETAILER = 'retailer';
+    const ROLE_SuperAdmin = 'super admin';
```

Comment:
Follow same naming convention. constants should be all capital like` const ROLE_ADMIN`

Reply:
Fixed